Assets/Scripts/Player/Development/ResourceEditor.cs

```diff
@@ -0,0 +1,137 @@
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+[CustomEditor(typeof(ResourceSpawner)), CanEditMultipleObjects]
+public class ResourceEditor : Editor {
+
+    private ResourceSpawner _resourceSpawner;
```

Comment:
Don't like the underscore convention to mark private fields, they add nothing of value